components/brave_wallet_ui/page/wallet_page.tsx

```diff
@@ -53,6 +54,10 @@ function App () {
     link.href = faveiconUrl
   }, [])
 
+  React.useEffect(() => {
+    removeDeprecatedLocalStorageKeys()
+  })
```

Comment:
Make sure you really want the behaviour that comes with this syntax: https://stackoverflow.com/a/58579462/1946230